### PR TITLE
[TrilinosApplication] Transition changes before supporting MPC in `TrilinosBlockBuilderAndSolver`

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Transition changes before supporting MPC in `TrilinosBlockBuilderAndSolver`

**🆕 Changelog**

- Transition changes before supporting MPC in `TrilinosBlockBuilderAndSolver`

### CODE REVIEW

File: applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h [hunk 1]
```diff
@@ -510,10 +505,12 @@ class TrilinosBlockBuilderAndSolver
...
         KRATOS_TRY
 
+        KRATOS_INFO_IF("TrilinosBlockBuilderAndSolver", BaseType::GetEchoLevel() > 2) << "Setting up the dofs" << std::endl;
+
         using DofsVectorType = Element::DofsVectorType;
-        
+
         // Gets the array of elements from the modeler
-        DofsVectorType dof_list;
+        DofsVectorType dof_list, second_dof_list;
```
Comment (sunethwarna) [inline applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h]: why do you need a `second_dof_list`? Is it used anywhere?

**Review by sunethwarna — COMMENTED** [review]
Thanks @loumalouomega . There is only one minor comment from my side.

File: applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h (hunk 1 above)
Comment (loumalouomega) [reply applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h]: In this case not, but will be use with the MPC, and this avoid conflicts, but I can remove it.

Comment (loumalouomega) [inline applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h]:
```suggestion
        DofsVectorType dof_list;
```

**Comment by sunethwarna** [comment]
Nice, small thing I noticed. May be a PR for the base branch is forgotten?

**Comment by loumalouomega** [comment]
> Nice, small thing I noticed. May be a PR for the base branch is forgotten?

No, the target branch, the PR https://github.com/KratosMultiphysics/Kratos/pull/10706 will be updated, andthe  final review will be there

**Comment by sunethwarna** [comment]
Oh I see, I do the other way usually, so the diffs are smaller in each PR :D.

**Comment by loumalouomega** [comment]
> Oh I see, I do the other way usually, so the diffs are smaller in each PR :D.

The problem is that there is an interdependency between both PR, and maybe I can think to set this merge with master if https://github.com/KratosMultiphysics/Kratos/pull/10706 is merged

**Pull request closed without merging**